### DISCUSSION TITLE
macos证书信任，还是不能打不开https，删除证书多次添加一样

### DISCUSSION THREAD

**Discussion by syymo** [1715173329/UnblockNeteaseMusic#705] [Q&A]
系统：MacOS bigsur 11.6.6
软件版本：版本2.3.8 (872)

<img width="1192" alt="image" src="https://user-images.githubusercontent.com/26571594/169634416-7d1a8178-0b27-4705-9294-a8f41073a035.png">

**Comment by pan93412** [comment]
在 Safari 放行，然後用 Touch ID/密碼驗證就可以用了。
